pkg/config/validation.go

```diff
@@ -299,6 +301,30 @@ func validateFairSharing(c *configapi.Configuration) field.ErrorList {
 	return allErrs
 }
 
+func validateAdmissionFairSharing(c *configapi.Configuration) field.ErrorList {
```

Comment:
Could you implement UTs in `validation_test.go`?

Reply:
Added